articles/synapse-analytics/overview-faq.yml

```diff
@@ -75,14 +75,19 @@ sections:
           Azure Synapse Link for SQL Server 2022 is in preview. For more information, see [What is Azure Synapse Link for SQL?](synapse-link/sql-synapse-link-overview.md).
 
       - question: |
-          Is Azure Synapse Link to Azure SQL Database generally available?
+          Does Azure Synapse workspace Support CI/CD? 
         answer: |
-          Azure Synapse Link for Azure SQL Database is in preview. For more information, see [What is Azure Synapse Link for SQL?](synapse-link/sql-synapse-link-overview.md).
+          Yes! All Pipeline artifacts, notebooks, SQL scripts, and Spark job definitions will reside in Git. All pool definitions will be stored in Git as Azure Resource Manager (ARM) templates. Dedicated SQL pool objects (schemas, tables, views, etc.) will be managed with database projects with CI/CD support.
 
       - question: |
-          Does Azure Synapse workspace Support CI/CD? 
+          What are the functional differences between dedicated SQL pools and serverless pools?
         answer: |
-          Yes! All Pipeline artifacts, notebooks, SQL scripts, and Spark job definitions will reside in Git. All pool definitions will be stored in Git as Azure Resource Manager (ARM) templates. Dedicated SQL pool objects (schemas, tables, views, etc.) will be managed with database projects with CI/CD support.
+          Capabilities and requirements are different between the two types of pools. Differences include database objects, query language capabilities, security, tools, data access, and data format. For detailed comparison of SQL pools and serverless pools, please visit [Pools Comparison](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql/overview-features). For best practices when using either type of pool, please view [Best Practices for Dedicated SQL Pool](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql/best-practices-dedicated-sql-pool) and [Best Practices for Serverless SQL Pool](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql/best-practices-serverless-sql-pool).
+
+      - question: |
+          What are Delta tables, and why should I use them?
```

Comment:
Could we add a section about how to use auto compaction in delta tables? In is one of the frequently asked questions

Reply:
Added in commit cb70304